### DISCUSSION TITLE
Sanity check - direction from one crystal frame in another crystal frame

### DISCUSSION THREAD

**Discussion by jadball** [pyxem/orix#513] [Q&A] [answered]
(note: I am using the orix fork mentioned in #442 to fix mean orientation issues)

Hello! I was wondering whether I was doing this correctly.
Let's say I have some grains of phase `Phase_B` embedded in a larger grain of `Phase_A`. I want to investigate whether there's an orientation relationship between them.
Would the following approach work to plot a pole figure of the B grains with respect to grain A?

```python
# what pole figure direction are we interested in?
pole_figure_direction = Miller(uvw=[1,1,1], phase=Phase_B)

# we have one Orix `Orientation` for grain A, and one Orix `Orientation` each for grains in B
B_direcs_in_A_frame = []
# iterate over grains in B
for orien_B in oriens_B:
    # get [111] crystal direction from grain B frame into lab frame
    B_direc_in_lab = ~orien_B * pole_figure_direction
    # rotate into the frame of the parent grain
    B_direc_in_A = orien_A * B_direc_in_lab
    B_direcs_in_A_frame.append(B_direc_in_A.data)

# then make a pole figure of B_direcs_in_A_frame...
```

Thanks in advance!
(oh, and thanks for your help with #468 , it worked!)

**Comment by hakonanes** [comment, marked as answer]
Hi James!

Yes, my understanding agrees with yours here. The transformations you stack should give you the crystal directions in grain A that are parallel with [111] in grain B. Do you get expected results?

**Reply by jadball** [reply]
Thank you!
It gave me some unexpected results for a specific material I was studying, but it turns out the real material OR is more complicated than we thought :) I also confirmed against MTEX with agrees with Orix here.
